File: bank-app/app/src/main/java/com/android/bankapp/service/BankService.java
package com.android.bankapp.service;

import com.android.bankapp.login.LoginRequest;
import com.android.bankapp.login.LoginResponse;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.POST;

public interface BankService {

    @POST("/api/login/")
    Call<LoginResponse> doLogin(@Body LoginRequest request);
}

File: Bank/app/src/main/java/com/jisellemartins/bank/service/BankService.kt
package com.jisellemartins.bank.service

import com.jisellemartins.bank.model.DetailsUser
import com.jisellemartins.bank.model.Login
import com.jisellemartins.bank.model.User
import retrofit2.Response
import retrofit2.http.Body
import retrofit2.http.GET
import retrofit2.http.POST
import retrofit2.http.Path

interface BankService {
    @POST("api/bank/login")
    suspend fun postLogin(@Body login: Login): Response<User>

    @GET("api/bank/login/{idUser}/statements")
    suspend fun getDetailsUser(@Path("idUser")idUser:String): Response<List<DetailsUser>>
}